Crane/app/src/main/java/androidx/compose/samples/crane/data/DestinationsLocalDataSource.kt

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.samples.crane.data
+
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private const val DEFAULT_IMAGE_WIDTH = "250"
+
+/**
+ * Annotated with Singleton as the class created a lot of objects.
+ */
+@Singleton
+class DestinationsLocalDataSource @Inject constructor() {
+
+    val craneRestaurants = listOf(
+        ExploreModel(
+            city = NAPLES,
+            description = "1286 Restaurants",
+            imageUrl = "https://images.unsplash.com/photo-1534308983496-4fabb1a015ee?ixlib=rb-1.2.1&auto=format&fit=crop&w=$DEFAULT_IMAGE_WIDTH"
```

Comment:
Don't bother now, but this can be improved so that the width is automatically applied from the actual size of the component. See Owl for an [example](https://github.com/android/compose-samples/blob/7da64ff119990df23bd5a026be8c2757ebbd2a82/Owl/app/src/main/java/com/example/owl/ui/utils/NetworkImage.kt#L87).

Reply:
Doing this in a follow-up PR! Thanks, didn't know about that

Reply:
Cool, I'm thinking about adding the `AmbientImageLoader` to Accompanist-Coil itself